gcsfs/tests/test_credentials.py

```diff
@@ -0,0 +1,9 @@
+from gcsfs.credentials import GoogleCredentials
+from gcsfs.tests.utils import my_vcr
+
+
+@my_vcr.use_cassette(match=["all"])
+def test_GoogleCredentials_None():
+    credentials = GoogleCredentials(project=None, token=None, access="read_only")
+    headers = {}
+    credentials.apply(headers)
```

Comment:
See that the headers contain something useful?

Reply:
In the circle tests they will be empty since there is no authentication with GCP. It's kind of a tricky integration to test without GCP since it relies on the value of an environmental variable.